### PR TITLE
Add orange color to step bar graph

### PR DESCRIPTION
**Pull request by bscheltinga**
fixes: https://github.com/PerfectFit-project/virtual-coach-issues/issues/433

### CODE REVIEW

File: Rasa_Bot/actions/helper.py [hunk 1]
```diff
@@ -1140,13 +1140,13 @@ def make_step_overview(date_array: List[str], step_array: List[int], step_goal:
          'steps': step_array,
          'goals': step_goal})
 
-    data['goal_achieved'] = data['steps'] >= data['goals']
+    data['goal_achieved'] = (data['steps'] >= 0.95*data['goals'])*1 + (data['steps'] >= data['goals'])*1
```
Comment (wbaccinelli) [inline Rasa_Bot/actions/helper.py:1143]:
```suggestion
    data['goal_achieved'] = (data['steps'] >= 0.95*data['goals']) + (data['steps'] >= data['goals'])*2
```
If I get it right, it should be like this, right?

**Comment by nelealbers** [comment]
@bscheltinga Have you checked if these colors are colorblind safe? I know standard red and green is usually an issue for about 10% of men. But there are shades of red and green that work (e.g., see [here](https://cran.r-project.org/web/packages/khroma/vignettes/tol.html)).

**Review by wbaccinelli — APPROVED** [review]
Now the users will never get confused anymore! Please, solve also the CI

**Comment by bscheltinga** [comment]
> @bscheltinga Have you checked if these colors are colorblind safe? I know standard red and green is usually an issue for about 10% of men. But there are shades of red and green that work (e.g., see [here](https://cran.r-project.org/web/packages/khroma/vignettes/tol.html)).

I always check all my images with a [colorblind simulator](https://www.color-blindness.com/coblis-color-blindness-simulator/):
image

For every dichromatic view, the difference between red and green is clear. Differences between green and orange not (see image). Feel free to adjust it, however, I think it (1) should be visually appealing and (2) the colours should speak for itself (red=bad, green=good).

File: Rasa_Bot/actions/helper.py (hunk 1 above)
Comment (bscheltinga) [reply Rasa_Bot/actions/helper.py:1143]: Nope, note that, if: _data['steps'] > data['goals'],_ it also satisfies _data['steps'] > 0.95*data['goals']_, meaning we will have 1+1 = 2

Comment (wbaccinelli) [reply Rasa_Bot/actions/helper.py]: Oh yes, I see

**Comment by nelealbers** [comment]
> > @bscheltinga Have you checked if these colors are colorblind safe? I know standard red and green is usually an issue for about 10% of men. But there are shades of red and green that work (e.g., see [here](https://cran.r-project.org/web/packages/khroma/vignettes/tol.html)).
> 
> I always check all my images with a [colorblind simulator](https://www.color-blindness.com/coblis-color-blindness-simulator/): image
> 
> For every dichromatic view, the difference between red and green is clear. Differences between green and orange not (see image). Feel free to adjust it, however, I think it (1) should be visually appealing and (2) the colours should speak for itself (red=bad, green=good).

Ah nice that red and green can be distinguished! Maybe worthwhile to adapt the orange shade then though? It 1 in every 10 men after all that won't be able to see a difference then.

**Pull request merged**